faunadb/query.go

```diff
@@ -357,6 +360,16 @@ func Epoch(num, unit interface{}) Expr { return fn2("epoch", num, "unit", unit)
 
 // Set
 
+// Singleton returns the history of the instance's presence of the provided ref.
```

Comment:
this description is a bit confusing. Is there a better way to state what `singleton` does?

Reply:
I wonder the same, I took this from a confluence describing the function